test/common-utils/alma-8.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Hmm, why don't I see test action runs for this PR? 😅  Taking a look!

Reply:
@jdputschadi Do you mind if I close and open this PR? I hope the action should run as expected post that.

Reply:
I don't mind at all.

Reply:
Looks like the test ran this time 🎉